src/docs/product/performance/apm-to-tracing.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Switching to @sentry/tracing"
+sidebar_order: 20
+---
+
+Our performance JavaScript SDK has been renamed from [`@sentry/apm`](https://www.npmjs.com/package/@sentry/apm) to [`@sentry/tracing`](https://www.npmjs.com/package/@sentry/tracing). The behaviour between `@sentry/apm` and `@sentry/tracing` is _identical_, but some imports and API have changed slightly.
+
+Although `@sentry/apm` is deprecated for `@sentry/tracing`, we will continue to support `@sentry/apm` with bug fixes for all `5.x.x` versions.
+
+## Migrating from @sentry/apm to @sentry/tracing
+
+For most cases, just replacing your `@sentry/tracing` import is enough to switch packages.
```

Comment:
Looking at the examples, it seems that more realistically at least 2 changes are required (for the majority of users):

```diff
 -import "@sentry/apm";
 +import "@sentry/tracing";
 ```

and

```diff
 Sentry.init({
    integrations: [
 -    new Integrations.Tracing(),
 +    new Integrations.BrowserTracing(),
    ]
  });
```


Suggestion:

Break this down into `Browser` and `Node`. For browser the change is as above or the new CDN snippet (replacing the `bundle.apm.min.js` one).

For Node.js, is the import change enough?

Reply:
Yes for Node the import change is enough. I guess if you were using Express, there is an added step. I will change.

Reply:
Wait, actually we should be fine because we didn't change the name for Express.